frontend/src/static/js/v_authorship.js

```diff
@@ -119,8 +138,12 @@ window.vAuthorship = {
     },
 
     hasCommits(info) {
-      if (window.REPOS[info.repo]) {
-        return window.REPOS[info.repo].commits.authorFinalContributionMap[info.author] > 0;
+      const { isMergeGroup, author } = info;
+      const repo = window.REPOS[info.repo];
+      if (repo) {
+        return isMergeGroup
+          ? Object.entries(repo.commits.authorFinalContributionMap).some(([name, cnt]) => name !== '-' && cnt > 0)
```

Comment:
A suggestion: we're comparing against the name `-` at several points.
Would it be more readable to wrap this in a method like `isUnknownAuthor` ?

Reply:
Yep that would be clearer. Perhaps it could be listed as a first time issue?

Reply:
Sure, I'll open an issue on it